app/src/main/res/values-pl/strings.xml

```diff
@@ -314,4 +314,72 @@
         • Przeprojektowano arkusz konfiguracji aplikacji dla lepszej użyteczności.\n
         • Ulepszono ekran sprawdzania kompatybilności z szczegółowymi informacjami o urządzeniu.
     </string>
+
+    <string name="privacy_title">100% Offline i Bezpieczne</string> 
+    <string name="privacy_desc">Hyper Bridge przetwarza wszystko bezpośrednio na procesorze Twojego telefonu.\n\nAplikacja NIE wymaga dostępu do internetu, co oznacza, że Twoje powiadomienia i dane osobiste nigdy nie opuszczają urządzenia.</string>
+    
+    <string name="language">Język</string>
+    <string name="language_desc">Zmień język aplikacji</string>
+    <string name="system_default">Domyślny systemowy</string>
+    <string name="save">Zapisz</string>
+    
+    <string name="backup_restore_title">Kopia zapasowa i Przywracanie</string>
+    <string name="backup_section_title">Zarządzanie Danymi</string>
+    <string name="backup_options_subtitle">Wybierz dane do uwzględnienia w operacji</string>
+    
+    <string name="option_app_settings">Ustawienia aplikacji</string>
+    <string name="option_app_settings_desc">Status konfiguracji, motyw, tryby działania</string>
+    
+    <string name="option_blocklist">Baza listy blokad</string>
+    <string name="option_blocklist_desc">Twoje zablokowane słowa kluczowe i aplikacje</string>
+    
+    <string name="option_priorities">Zasady Priorytetów</string>
+    <string name="option_priorities_desc">Niestandardowe konfiguracje priorytetów powiadomień</string>
+    
+    <string name="action_export">Utwórz kopię zapasową</string>
+    <string name="action_import">Przywróć kopię zapasową</string>
+    
+    <string name="export_success">Kopia zapasowa została zapisana pomyślnie</string>
+    <string name="export_failed">Błąd eksportu: %s</string>
+    <string name="import_success">Dane zostały pomyślnie przywrócone</string>
+    <string name="import_failed">Błąd importu: %s</string>
+    <string name="invalid_backup_file">Nieprawidłowy plik kopii zapasowej HyperBridge</string>
+    <string name="db_version_mismatch">Ostrzeżenie: Kopia zapasowa pochodzi z innej wersji bazy danych</string>
+    
+    <string name="autostart_manual_check">Nie można automatycznie zweryfikować statusu.\nProszę sprawdzić to ręcznie, aby zapewnić prawidłowe działanie usługi</string>
```

Comment:
"Proszę sprawdzić to ręcznie, aby zapewnić prawidłowe działanie usługi"
IMO better to replace it with
"Aby usługa dzialała poprawnie, sprawdź go ręcznie"